binder/requirements.txt

```diff
@@ -3,3 +3,4 @@ scikit-learn==0.22.2.post1
 numpy==1.18.5
 nyoka==4.3.0
 matplotlib==3.3.2
+sklearn
```

Comment:
Please remove this change, sklearn is already included (`scikit-learn`) while this other package is not intended to be used: see [pip website](https://pypi.org/project/sklearn/)